File: GestionInterim/app/src/main/java/com/example/gestioninterim/InscriptionActivity.java
package com.example.gestioninterim;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import com.example.gestioninterim.slides.FragmentSlideParent;

public class InscriptionActivity extends AppCompatActivity {

    FragmentManager fragmentManager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inscription);

        fragmentManager = getSupportFragmentManager();

        FragmentTransaction fragmentTransaction = fragmentManager.beginTransaction();

        FragmentTitleSubTitle fragment = new FragmentTitleSubTitle();

        fragmentTransaction.add(R.id.title_subtitle_slogan, fragment);

        fragmentTransaction.commit();
    }

}

File: GestionInterim/app/src/main/java/com/example/gestioninterim/InscriptionActivity.kt
package com.example.gestioninterim

import android.os.Bundle
import android.widget.CheckBox
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.FragmentManager

class InscriptionActivity : AppCompatActivity() {

    var fragmentManager: FragmentManager? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_inscription)

        fragmentManager = supportFragmentManager
        val fragmentTransaction = fragmentManager!!.beginTransaction()
        val fragment = FragmentChooseInscription()
        fragmentTransaction.add(R.id.containerChooseInscription, fragment)
        fragmentTransaction.commit()

    }
}